bin/prettier.js

```diff
@@ -68,7 +68,7 @@ const argv = minimist(args, {
     version: "v",
     "list-different": "l"
   },
-  unknown: param => {
+  unknown: (param) => {
```

Comment:
Have you added all these parens to Prettier's source code manually? You do know that we run Prettier 1.5.2 on all source files, right? So this will cause lint errors...

Reply:
Hah, I did not know 😄 that makes things simpler. Thanks @lydell (I'd run `bin/prettier.js`, assuming that was what we do – might be worth adding something to `scripts`)

Reply:
Thanks! The diff is much easier to read now: :+1:

I've configured my editor (Atom) to run `eslint --fix` on save, so this has never happened to me ... (we run Prettier inside ESLint using eslint-plugin-prettier)

Reply:
Thanks! The diff is much easier to read now: :+1:

I've configured my editor (Atom) to run `eslint --fix` on save, so this has never happened to me ... (we run Prettier inside ESLint using eslint-plugin-prettier)

Reply:
For reference, [here](https://github.com/prettier/prettier/pull/1634#issuecomment-302945967) is where we decided to dogfood the latest published version of prettier, rather than the current commit.